### PR TITLE
Improve Admin Navbar Look

### LINKED ISSUE #263: Redesign AdminNavbar UI
As of now, our navbar looks like this:
image

It's bland, and we can improve the admin's UI. To mess with the styling, write changes in [`src/Components/Navbar/navbar.css`](https://github.com/SCE-Development/Core-v4/blob/dev/src/Components/Navbar/navbar.css)

The admin navbar can be found in [`src/Components/Navbar/AdminNavbar.js`](https://github.com/SCE-Development/Core-v4/blob/dev/src/Components/Navbar/AdminNavbar.js)

### PR DESCRIPTION
**Pull request by surabhigupta2029**
Improved the look of the Admin page navbar

Resolves #263
navbar

### CODE REVIEW

File: src/Components/Navbar/navbar.css [hunk 1]
```diff
@@ -9,13 +9,30 @@
   margin-bottom: 5px;
 }
 
+<<<<<<< HEAD
```
Comment (evanugarte) [inline src/Components/Navbar/navbar.css:12]: remove this line

File: src/Components/Navbar/navbar.css [hunk 2]
```diff
@@ -9,13 +9,30 @@
...
+  color: #aee6eb;
   transition: color 0.2s;
+  background-color: rgb(13, 70, 83);
+
+
 }
 .nav-link:hover {
   color: #FF0000;
 }
+=======
```
Comment (evanugarte) [inline src/Components/Navbar/navbar.css:23]: here too

File: src/Components/Navbar/navbar.css [hunk 3]
```diff
@@ -9,13 +9,30 @@
...
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+  color: #308f97;
+  background-color:#648cc0;
+} 
+
+ .nav-admin-link{
+  color: white!important;
+  fill: white;
+} 
+>>>>>>> updating branch
```
Comment (evanugarte) [inline src/Components/Navbar/navbar.css:35]: same here

File: src/Pages/Overview/Overview.js [hunk 4]
```diff
@@ -125,9 +125,8 @@ export default class OverviewBoard extends Component {
 
   render() {
     return (
-      <div className='layout'>
-        <h1>Users Dashboard</h1>
-
+      <div className='layout' >
```
Comment (evanugarte) [inline src/Pages/Overview/Overview.js:128]: this file shouldnt be changed at all, can you refert it back to `dev`? 

do:
`git checkout origin/dev [filename]`

File: src/Components/Navbar/AdminNavbar.js [hunk 5]
```diff
@@ -21,10 +21,10 @@ export default function AdminNavbar(props) {
   ];
 
   return (
-    <Navbar light className='admin-nav navbar-expand-md'>
+    <Navbar dark className='admin-nav navbar-expand-md'>
       <NavbarBrand href='/'>
-        Software & Computer Engineering Society
-      </NavbarBrand>
+        Admin Page
```
Comment (evanugarte) [inline src/Components/Navbar/AdminNavbar.js:26]: Do we wanna call the Admin side `Admin Page` now? Its more of a collection of pages

File: api/routes/InventoryItem.js [hunk 6]
```diff
@@ -14,8 +14,10 @@ const {
   FORBIDDEN,
   NOT_FOUND
```
Comment (evanugarte) [inline api/routes/InventoryItem.js:15]: this file shouldnt be changed at all, can you refert it back to `dev`? 

do:
`git checkout origin/dev [filename]`

File: api/routes/InventoryItem.js [hunk 7]
```diff
@@ -14,8 +14,10 @@ const {
...
 } = require('../constants').STATUS_CODES;
+
 const addErrorLog = require('../util/errorLog');
 
+
```
Comment (seemavora) [inline api/routes/InventoryItem.js:20]: remove blank space

File: src/Components/Navbar/navbar.css [hunk 8]
```diff
@@ -8,14 +8,39 @@
...
+  background-color:#648cc0;
+}
 
 .admin-nav, .nav-link {
-  color: #515151;
+  color: #aee6eb;
   transition: color 0.2s;
+  background-color: rgb(13, 70, 83);
+
+
```
Comment (seemavora) [inline src/Components/Navbar/navbar.css:23]: remove blank space :)

File: src/Pages/Overview/Overview.css [hunk 9]
```diff
@@ -7,24 +7,26 @@
 
 .layout{
   margin: 10px auto;
-  width:700px
+  width:700px;
 }
 
 h1{
```
Comment (aashrav) [inline src/Pages/Overview/Overview.css:13]: Don't style h1 tag. Give target element a className and then style it

File: api/routes/InventoryItem.js (hunk 6 above)
Comment (evanugarte) [reply api/routes/InventoryItem.js:15]: This isnt resolved, please double check before

**Pull request merged**